packages/api/indexer/processor/hustler.go

```diff
@@ -88,8 +91,12 @@ func (p *HustlerProcessor) Setup(address common.Address, eth interface {
 }
 
 func (p *HustlerProcessor) ProcessAddRles(ctx context.Context, e bindings.HustlerAddRles) (func(tx *ent.Tx) error, error) {
-	ctx, log := logger.LogFor(ctx)
-	log.Debug().Str("indexer", "HUSTLER").Str("indexer", "HUSTLER").Msgf("ProcessAddRles")
+	// Tagged logger
+	ctx, log := logger.LogFor(
+		ctx,
+		func(zctx *zerolog.Context) zerolog.Context {
+			return zctx.Str("HustlerProcessor", "ProcessAddRles")
```

Comment:
Can you hadd the hustlerId here please, so we know to whom the subsequent logs belong

Reply:
ProcessAddRles adds new bodyparts to the contract itself, not individual hustlers!